source/uwp/AdaptiveCardTestApp/AdaptiveCardTestApp.csproj

```diff
@@ -149,8 +149,86 @@
     <Content Include="..\..\..\samples\v1.1\Elements\*.json">
       <Link>LinkedCards\v1.1\Elements\%(FileName)</Link>
     </Content>
```

Comment:
Revert this change. It's probably a result of deleting the samples, but we don't want these listed explicitly, we want to include the folder.

Reply:
thanks

---
In reply to: [287427936](https://github.com/microsoft/AdaptiveCards/pull/2970#discussion_r287427936) [](ancestors = 287427936)